_includes/page-header.html

```diff
@@ -7,8 +7,8 @@ <h1 class="page__title" itemprop="name headline">{{ page.title | escape }}</h1>
   {%- endif -%}
 
   {% assign content = content | strip_newlines %}
-  {%- if content != '' -%}
-  <div class="page__desc">{{ content }}</div>
+  {%- if content != '' and page.permalink !== '/about/'-%}
```

Comment:
I would change this to make it more generic as it applies to all pages*, not just the about page.
```suggestion:
{%- if content != '' and page.layout != 'page' -%}
```

* Pages in this instance means individual pages like the about page, not the "pages" that are the archive listings.